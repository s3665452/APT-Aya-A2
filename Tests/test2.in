LRBUYRBLLURUBYUBBLUUBLYYYURLURRRLBBUYBRBULRYBYLLLLYLRBBYLYLRUURRBBYRYYBULULLLUBBUYRRYUYYURYLRRURBYUB
Arya
Sansa
turn 4 Y 4
turn 3 L 3
turn 1 R 1
turn 0 R 5
turn 2 B 2
turn 5 U 4
turn 0 U 5
turn 0 L 3
