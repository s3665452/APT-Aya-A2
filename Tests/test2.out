Factories:
0: R R R R
1: F B R Y
2: R U B B 
3:  
4: 
5: U U U U U

Score for Player A: 0
Mosaic for Arya:
1:         . || . . . . .
2:       . . || . . . . .
3:     . . . || . . . . .
4:   Y Y Y Y || . . . . .
5: . . . . . || . . . . .
broken:

Score for Player B: 0
Mosaic for Sansa:
1:         . || . . . . .
2:       . . || . . . . .
3:     L L L || . . . . .
4:   . . . . || . . . . .
5: . . . . . || . . . . .
broken:

Factories:
0:  
1: 
2: R U B B 
3:  
4: 
5: U U U U U

Score for Player A: 0
Mosaic for Arya:
1:         R || . . . . .
2:       . . || . . . . .
3:     . . . || . . . . .
4:   Y Y Y Y || . . . . .
5: R R R R . || . . . . .
broken:

Score for Player B: 0
Mosaic for Sansa:
1:         R || . . . . .
2:       . . || . . . . .
3:     L L L || . . . . .
4:   . . . . || . . . . .
5: . . . . . || . . . . .
broken:

Factories:
0:  
1: 
2:  
3:  
4: 
5:  

Score for Player A: 0
Mosaic for Arya:
1:         . || . . . . .
2:       B B || . . . . .
3:     . . . || . . . . .
4:   Y Y Y Y || . . . . .
5: . . . . . || . . . . .
broken:

Score for Player B: 0
Mosaic for Sansa:
1:         R || . . . . .
2:       . . || . . . . .
3:     L L L || . . . . .
4:   U U U U || . . . . .
5: . . . . . || . . . . .
broken:

Factories:
0: U U U U U
1: R R B U
2: R Y B L 
3: L L L B
4: B B Y L
5: B U Y R  
Score for Player A: 0
Mosaic for Arya:
1:         . || . . . . .
2:       B B || . . . . .
3:     . . . || . . . . .
4:   Y Y Y Y || . . . . .
5: U U U U U || . . . . .
broken:

Score for Player B: 0
Mosaic for Sansa:
1:         R || . . . . .
2:       . . || . . . . .
3:     L L L || . . . . .
4:   U U U U || . . . . .
5: . . . . . || . . . . .
broken: L L L
